pkg/logstructured/sqllog/sql.go

```diff
@@ -266,10 +266,12 @@ func (s *SQLLog) startWatch() (chan interface{}, error) {
 
 func (s *SQLLog) poll(result chan interface{}) {
 	var (
-		last int64
+		last     int64
+		skip     int64
+		skipTime time.Time
 	)
 
-	wait := time.NewTicker(120 * time.Second)
+	wait := time.NewTicker(time.Second)
```

Comment:
this reminds of the previous discussion about making the value configurable.. but 1 second sounds like a good magic number to me for now